server/routes/auth_routes.py

```
@@ -146,6 +146,12 @@ def register():
                 except Exception as policy_err:
                     logging.warning("Failed to seed command policy for org %s", org_id, exc_info=policy_err)
 
+                try:
+                    from utils.auth.tool_registry import seed_org_tool_permissions
+                    seed_org_tool_permissions(org_id, user_id)
+                except Exception as tool_perm_err:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)